roles/preflight-checks/tasks/ceph.yml

```diff
@@ -35,3 +35,14 @@
   delegate_to: "{{ groups['ceph_monitors'][0] }}"
   delegate_facts: yes
   when: result_osdhosts.stdout|int < groups['ceph_osds']|length
+
+- name: fail if current ceph version is less than Jewel
+  shell: test $(ceph --version |grep -oP "\d+\.\d+\.?\d{0,3}") \>
+         "{{ preflight_checks.ceph_version_larger_than }}"
+  delegate_to: "{{ item }}" 
+  delegate_facts: True
+  with_items: 
+    - "{{ groups['ceph_monitors'] }}"
```

Comment:
we need a default value in case groups['ceph_monitors'] is undefined
groups['ceph_monitors']|default([])
groups['ceph_osds']|default([])